src/routing_message_filter.rs

```diff
@@ -70,9 +72,15 @@ impl RoutingMessageFilter {
 
     // Filter outgoing `RoutingMessage`. Return whether this specific message has been seen recently
     // (and thus should not be sent, due to deduplication).
+    //
+    // Return `false` if serialisation of the message fails - that can be handled elsewhere.
```

Comment:
seems the previous definition of the return value is whether the msg has been inserted previously.

hence a normal meaning of `false` is that this is the first time of insertion
is it really OK to change the definition?